srcpkgs/nudoku/template

```diff
@@ -1,17 +1,23 @@
 # Template file for 'nudoku'
 pkgname=nudoku
-version=1.0.0
+version=2.0.0
 revision=1
 build_style=gnu-configure
-hostmakedepends="automake"
+hostmakedepends="automake pkg-config tar xz gettext gettext-devel"
 makedepends="ncurses-devel"
 short_desc="Ncurses based sudoku game"
 maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/jubalh/nudoku"
 distfiles="https://github.com/jubalh/${pkgname}/archive/${version}.tar.gz"
-checksum=80fb9996c28642920951c20cfd5ca6e370d75240255bc6f11067ae68b6e44eca
+checksum=44d3ec1ff34a010910ac7a92f6d84e8a7a4678a966999b7be27d224609ae54e1
+
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	configure_args+="--disable-nls"
```

Comment:
missing space, should be `configure_args+=" --disable-nls"` in order to properly handle concat in case other configure args are added

Reply:
ok, done.